integration-tests/support/pages/tabs/PipelinerunsTabPage.ts

```diff
@@ -47,6 +47,51 @@ export class DetailsTab {
         }
       });
   }
+
+  static clickOnNode(nodeId: string) {
+    cy.get(pipelinerunsTabPO.node(nodeId)).click();
+  }
+
+  static checkNodeDrawerPanelResult(section: string, value: string) {
+    cy.contains(pipelinerunsTabPO.PF4TableRow, section)
+      .contains(value)
+      .scrollIntoView()
+      .should('be.visible');
+  }
+
+  static checkVulScanOnClairDrawer() {
+    cy.get(pipelinerunsTabPO.drawerPanel)
+      .contains(pipelinerunsTabPO.listGroup, 'Vulnerabilities scan')
+      .contains(/Critical(\d+).*High(\d+).*Medium(\d+).*Low(\d+)/g)
```

Comment:
This is used in 3 places, would it make sense to extract it to the variable? E.g. in PO file?

Reply:
I don't think we should move it to PO file as its not locator instead we can store it in var in main test and then pass it as parameter.

Reply:
Updated. moved to variable.